content/bibs/pubs.bib

```diff
@@ -2847,6 +2865,15 @@ @inproceedings{pedigo_naisys_2020
   keywords = {abspos}
 }
 
```

Comment:
the change wasn't reflected b/c the wrong bib file was being updated. It's confusing (I know!), so if you don't want any hassle, you can simply create an issue and I'll take care of it for you (if you want). Otherwise everything looks great!

Reply:
im confused, it looks like you already did?